oreilly-scenarios/cockroach-with-friendpay/configure-environment.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+wget -qO- https://binaries.cockroachdb.com/cockroach-v21.1.1.linux-amd64.tgz | tar -xz
+cp -i cockroach-v21.1.1.linux-amd64/cockroach /usr/local/bin/
+mkdir -p /usr/local/lib/cockroach
+cp -i cockroach-v21.1.1.linux-amd64/lib/libgeos.so /usr/local/lib/cockroach/
```

Comment:
Do you need the geoindex functionality? Does the example run without it? Probably not worth taking out.